### PR TITLE
Migrate links

### PR DESCRIPTION
**Pull request by dougdroper**
Rake task to Represent the content item downstream without state changes.

Use the expanded link set presenter for links that are ready to migrate. (currently service_manual)

Use `links` for and second level expansion instead of `expanded_links` so that there are no changes to frontend apps. 

https://trello.com/c/yNL1Lziv/727-prepare-formats-to-switch-to-expanded-links

### CODE REVIEW

File: lib/tasks/migrate_links.rake [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  Usage
+  rake 'migrate_links:format[:schema_name]'
+  """
+  task :format, [:schema_name] => :environment do |_t, args|
+    schema_name = args[:schema_name]
+
+    raise "Schema name needs to be in MigrateExpandedLinks" unless Presenters::MigrateExpandedLinks.document_types.include?(schema_name)
+
+    Commands::V2::RepresentDownstream.new.call(
+      ContentItem.where(document_type: schema_name)
```
Comment (dougdroper) [inline lib/tasks/migrate_links.rake]: change to schema_name

File: app/commands/v2/represent_downstream.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        draft_content_items = Queries::GetLatest.call(
+          filter.filter(state: %w{draft published}))
+        live_content_items = filter.filter(state: "published")
+
+        draft_content_items.each do |draft_content_item|
+          send_to_content_store(draft_content_item, Adapters::DraftContentStore)
+        end
+
+        live_content_items.each do |live_content_item|
```
Comment (dougdroper) [inline app/commands/v2/represent_downstream.rb]: find_each

File: app/commands/v2/represent_downstream.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+      def call(scope)
+        filter = ContentItemFilter.new(scope: scope)
+
+        draft_content_items(filter).find_each do |draft_content_item|
+          send_to_content_store(draft_content_item, Adapters::DraftContentStore)
+        end
+
+        live_content_items(filter).find_each do |live_content_item|
+          send_to_content_store(live_content_item, Adapters::ContentStore)
```
Comment (danielroseman) [inline app/commands/v2/represent_downstream.rb]: Should we not also re-send live content items to the draft content-store? (If so this should probably happen before the draft_content_items loop.)

Comment (danielroseman) [inline app/commands/v2/represent_downstream.rb]: Ah ignore me, I've just seen that draft_content_items includes live ones. Maybe rename to something like `items_for_live_store`?

**Comment by danielroseman** [comment]
Apart from the one name change, :+1:

**Pull request merged**